DependencyInjection/Configuration.php

```diff
@@ -165,6 +165,7 @@ public function getConfigTreeBuilder()
                     ->prototype('scalar')->end()
                 ->end()
                 ->scalarNode('target_path_parameter')->defaultNull()->end()
+                ->booleanNode('use_twig_runtime_extension')->defaultFalse()->end()
```

Comment:
added this to keep BC as (for whatever reason) someone might call public methods provided by the deprecated extension class.